src/app/core/contentful.service.ts

```diff
@@ -44,11 +50,14 @@ export class ContentfulService {
 
   // get landing page events metadata from backend
   async getEvents(): Promise<any> {
-    return this.http.get(`${environment.apiUrl}/events`).toPromise();
+    return this.http.get(`${environment.apiUrl}/events`).toPromise().catch(
+      (error: any) => {
+        console.log(error);
+      });
```

Comment:
You should make helper function `logError` to handle error logging logic and change all these catch instances to use it.

Also, should we `console.error` instead of `console.log`, or is there a reason to use log? `console.debug` is also an option or `warn`.